dev-libs/json_c/json_c-0.13.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="A JSON implementation in C"
+DESCRIPTION="JSON-C implements a reference counting object model that allows \
+you to easily construct JSON objects in C, output them as JSON formatted \
+strings and parse JSON formatted strings back into the C representation of \
+JSON objects."
+HOMEPAGE="https://github.com/json-c/json-c/wiki"
+COPYRIGHT="2009-2012 Eric Haszlakiewicz
+	2004, 2005 Metaparadigm Pte Ltd"
```

Comment:
A bit of grep sauce and here's what we've got

List of copyright holders and their associated years:
- 2009-2012, 2016-2017 Eric Haszlakiewicz
- 2004-2005, 2013 Metaparadigm Pte. Ltd.
- 2009-2012 Hewlett-Packard Development Company, L.P.
- 2008-2009 Yahoo! Inc.
- 2016 Alexandru Ardelean.

For the HTML docs if it's installed:
- 2011 John Resig
- 2011 The Dojo Foundation
- 2011 jQuery authors
- 2010 "Cowboy" Ben Alman
- 2013 Steven Benner
- 2011-2014 Dave Furfero